app.yaml

```diff
@@ -2,4 +2,5 @@ runtime: python37
 entrypoint: gunicorn -b :$PORT main:app
 
 env_variables:
-  FOURSQUARE_DATA_VERSION: "20200503-v0"
+  FOURSQUARE_DATA_VERSION: "20200504-v0"
+  BUCKET_NAME: "vd-weather-data"
```

Comment:
Let's add a `WEATHERAPI_DATA_VERSION` variable to set the as of date for the weather data as part of the deployment. We can use the same bucket (`data.visitdata.org`).